### PR TITLE
[bug](backup) Add table id into convert_rowset_ids()

### PR DESCRIPTION
**Pull request by xy720**
## Proposed changes

Issue Number: step 1 in version master in #37076

### CODE REVIEW

**Comment by xy720** [comment]
run buildall

File: gensrc/thrift/AgentService.thrift [hunk 1]
```diff
@@ -309,6 +309,7 @@ struct TCloneReq {
     10: optional i32 timeout_s;
     11: optional Types.TReplicaId replica_id = 0
     12: optional i64 partition_id
+    13: optional i64 table_id
```
Comment (cambyzju) [inline gensrc/thrift/AgentService.thrift:312]:
```suggestion
    13: optional i64 table_id = 0
```

Comment (xy720) [reply gensrc/thrift/AgentService.thrift]: changed to -1, may be better.

**Comment by xy720** [comment]
run buildall

**Comment by xy720** [comment]
run p0

**Review by cambyzju — APPROVED** [review]

**Review by lide-reed — APPROVED** [review]
LGTM

**Pull request merged**